draft-ietf-rats-architecture.md

```diff
@@ -973,28 +973,40 @@ A more detailed discussion with examples appears in {{time-considerations}}.
 
 # Privacy Considerations
 
-The conveyance of Evidence and the resulting Attestation Results
-reveal a great deal of information about the internal state of a
-device.  In many cases, the whole point of the Attestation process is
-to provide reliable information about the type of the device and the
-firmware/software that the device is running.  This information might be
-particularly interesting to many attackers. For example, knowing that a device is
-running a weak version of firmware provides a way to aim attacks better.
-
-Evidence and Attestation Results data structures are expected to support
-integrity protection encoding (e.g., COSE, JOSE, X.509) and optionally might
-support confidentiality protection (e.g., COSE, JOSE).
-Therefore, if confidentiality protection is omitted or unavailable, the protocols
-that convey Evidence or Attestation Results are responsible for detailing what
-kinds of information are disclosed, and to whom they are exposed.
-
-Furthermore, because Evidence might contain sensitive information,
-Attesters are responsible for only sending such Evidence to trusted
-Verifiers.  Some Attesters might want a stronger level of assurance of
-the trustworthiness of a Verifier before sending Evidence to it.  In such cases,
-an Attester can first act as a Relying Party and ask for the Verifier's own
-Attestation Result, and appraising it just as a Relying Party would appraise
-an Attestation Result for any other purpose.
+Every claim in Attestation Evidence and Attestation Results is potentially PII (Personally Identifying Information) depending on the end-end use case of the attestation.
+Each deployment of attestation must take into account how each claim may or may not be used in a privacy violating manner by the relying parties receiving the attestation.
+
+In cases where the device sending attestation is owned by the relying party, little to no consideration for privacy is needed. In other cases, such as when a mobile phone is used to access a large number of web sites, apps and services, privacy must be considered very thoroughly.
```

Comment:
```suggestion
In cases where the Attester sending Evidence is owned by the Relying Party Owner, little to no consideration for privacy is needed. In other cases, such as when a mobile phone is used to access a large number of web sites, apps and services, privacy must be considered very thoroughly.
```

Reply:
Not entirely sure that this suggested change is not breaking the intent of the original text.

Reply:
"In cases where Evidence is owned by the Relying Party Owner, consideration for privacy may be of lesser importance. In other cases where Evidence is owned by the Attester, such as when a mobile phone is used to access a large number of web sites, applications or services, consideration for privacy may be of greater importance."